pegasus/sites.v3/code.org/public/edit-csp-table/splat.haml

```diff
@@ -127,15 +156,23 @@ angular: recordsApp
     }
 
   }]);
-
+  function makeColumn(name) {
+    return {
+      id: name,
```

Comment:
The duplicate state is a little confusing. I get that you need them for rename, but it might be more clear to have one field `name` that is used in most places, and another field `old_name` or `new_name` that is only used during rename.